docs/wiki/The CI environment - Publishing.md

```diff
@@ -61,10 +61,10 @@ The names of published modules differ slighly depending on the location they are
 **Examples**
 
 - Vaults
-   - Before: `modules\Microsoft.RecoveryServices\vaults`
+   - Before: `modules\RecoveryServices\vaults`
    - After: `bicep/modules/microsoft.recoveryservices.vaults`
 - ReplicationProtectionContainerMappings
-   - Before: `modules\Microsoft.RecoveryServices\vaults\replicationFabrics\replicationProtectionContainers\replicationProtectionContainerMappings`
+   - Before: `modules\RecoveryServices\vaults\replicationFabrics\replicationProtectionContainers\replicationProtectionContainerMappings`
    - After: `bicep/modules/microsoft.recoveryservices.vaults.replicationfabrics.replicationprotectioncontainers.replicationprotectioncontainermappings`
```

Comment:
I guess this changes too? There are a few references

Reply:
removed what I could find 👀